Stack/Opc.Ua.Core/Stack/State/NodeState.cs

```diff
@@ -66,17 +66,20 @@ public virtual object Clone()
         /// </summary>
         protected object CloneChildren(NodeState clone)
         {
-            if (m_children != null)
+            lock(m_children)
```

Comment:
shouldn't this be childrenLock?

Reply:
Yes. Good catch. I'll fix it.

Reply:
I made some changes to this CloneChildren method since I noticed it called the Clone inside the lock which again takes the m_childLock of another NodeState. I changed it so that Clone is not called inside a lock.